docs/pages/guides/apple-privacy.mdx

```diff
@@ -0,0 +1,62 @@
+
+Those APIs currently include accessing UserDefaults, file timestamp APIs, system boot time APIs, disk space APIs and active keyboard APIs, but Apple considers it an open list that can be expanded in the future.
+
+## Including an iOS Privacy Manifest in Expo Managed Apps
+
+If you're using an Expo managed app, you can include an iOS privacy manifest by adding a `privacyManifests` field to your `app.json` file, under the `expo.ios` keys. This requires Expo SDK 50 or later.
+
+Here's an example of how you can include a privacy manifest in your `app.json` file:
+
+```json
+{
+  "expo": {
+    "name": "My App",
+    "slug": "my-app",
+    // other fields...
+    "ios": {
+      "privacyManifests": {
+        "NSPrivacyAccessedAPITypes": [
+          {
+            "NSPrivacyAccessedAPIType": "NSPrivacyAccessedAPICategoryUserDefaults",
+            "NSPrivacyAccessedAPITypeReasons": ["CA92.1"]
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+You can identify the `NSPrivacyAccessedAPITypes` and `NSPrivacyAccessedAPITypeReasons` values by looking at the [Apple Developer Documentation](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api).
+
+> Warning:
+> Make sure you have updated your Expo SDK packages to latest versions for your SDK version using `npx expo install --fix`.
+
+### Including required reasons for Expo SDK packages and other 3rd party libraries
+
+As of now, Apple does not correcly parse all the `PrivacyInfo` files included by static cocoapods dependencies (like Expo SDK packages and other ecosystem packages). This means that you may need to include the required reasons for the APIs used by those dependencies in your app's `PrivacyInfo.xcprivacy` file or the configuration in your `app.json`.
+
+All Expo SDK packages that use "required reason" APIs file have a `PrivacyInfo` file included in the package directory. [Here's an example privacy info file]](https://github.com/expo/expo/blob/main/packages/expo-application/ios/PrivacyInfo.xcprivacy) included with the `expo-application` package.
+
+You can usually identify the required reasons for the APIs used by other 3rd party libraries by checking if the package you intend to use has a `PrivacyInfo.xcprivacy` file in the `node_modules/package_name/ios` directory. If it does, you can check the `NSPrivacyAccessedAPITypes` and `NSPrivacyAccessedAPITypeReasons` values in that file and copy those values to your configuration.
+
+As an alternative, Apple notifies developers after they submit a build with missing privacy manifest files or specific reasons, so you can wait until you receive a notification email from Apple and then include the required reasons listed in the email in your app's `PrivacyInfo.xcprivacy` file or the configuration in your `app.json` file.
+
+## Including an iOS Privacy Manifest in Bare Expo Apps
```

Comment:
maybe we can follow this pattern? https://docs.expo.dev/versions/latest/sdk/camera/#are-you-using-this-library-in-a